src/lib.rs

```diff
@@ -18,9 +18,10 @@ pub mod style;
 mod algo;
 mod forest;
 mod id;
-
 mod sys;
 
+use core::fmt::{Display, Formatter, Result};
```

Comment:
These imports also need to be gated under #[cfg(feature = "std")].

Reply:
Why is this the case? Isn't the `core` crate available in a `no_std` context?

Reply:
Is this a case of `Display`, `Formatter` and `Result` needs to be re-exported under `mode core { ... }` in `sys.rs`?